test/integration/setup/network.js

```diff
@@ -22,7 +22,7 @@ var utils = require('../utils');
 
 module.exports = {
 	waitForAllNodesToBeReady(configurations, cb) {
-		const retries = 20;
+		const retries = 40;
```

Comment:
Why we need to increase that?

Reply:
This was messed up from me. I staged some changes, but whole file was committed.